proxies/live/apiproxy/policies/AssignMessage.Swap.CorrelationHeader.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.Swap.CorrelationHeader">
+  <Set>
+    <Headers>
+      <Header name="nhsd-correlation-id">{message.header.X-Correlation-ID}.{messageid}</Header>
```

Comment:
Should the property be lower case too? Also the header in the remove section below?

It shouldn't make a difference but I guess we should be consistent?